recipes-dotnet/coreclr/coreclr/toolchain.patch

```diff
@@ -1,8 +1,18 @@
+From 51d254e8993202a54c160ba52b412601090bd2cb Mon Sep 17 00:00:00 2001
```

Comment:
Used devtool for editing, it added some extra cruft. Can remove that if you want to..